test/functional/fixtures/regression/t174562/test.js

```diff
@@ -0,0 +1,3 @@
+it('T174562 - jQuery UI selectmenu demo', function () {
```

Comment:
The same. You should leave the original ticket name, or put a short description of the problem to the test name.
`jQuery UI selectmenu demo` says nothing. You can call it:
`Click on jQuery UI popup item is out of popup bounds (T174562)`